kart.spec

```diff
@@ -115,6 +115,7 @@ pyi_analysis = Analysis(
         '_cffi_backend',
         # via a cython module ???
         'csv',
+        'shellingham.posix',
```

Comment:
is there a `shellingham.nt` module we should be adding on windows? 
On windows we may be running in git-bash/mingw-sh, so I guess it needs both?

Reply:
Not sure on that one - does pyinstaller include things in the module but not submodules? e.g. shellingham.nt is not a submodule. I'm not clear on that.

Reply:
i was just guessing, since pyinstaller's autodetection code clearly didn't find the posix module

Reply:
the auto-detection finds import statements, but doesn't deal well with clever conditional imports inside functions.

Reply:
I've added 'shellingham.nt' as it fails when I added an e2e powershell test.

```
ModuleNotFoundError: No module named 'shellingham.nt'
```